ha/util/conf_store.py

```diff
@@ -110,3 +121,93 @@ def set_cluster_cardinality(self, index):
+        """
+        try:
+            cvg_count = Conf.get(index, GconfKeys.CVG_COUNT.value.replace("(node_id)", node_id))
+            disk_list = []
+            for cvg_index in range(cvg_count):
+                num_of_data_disks = Conf.get(index, GconfKeys.DATA_COUNT.value.replace("(node_id)", node_id)
+                                            .replace("(cvg_index)", str(cvg_index)))
+                num_of_metadata_disks = Conf.get(index, GconfKeys.METADATA_COUNT.value.replace("(node_id)", node_id)
+                                                .replace("(cvg_index)", str(cvg_index)))
+
+                for d_index in range(num_of_data_disks):
+                    disk_list.append(Conf.get(index, GconfKeys.DATA_DISK.value.replace("(node_id)", node_id).
+                                              replace("(cvg_index)", str(cvg_index)).replace("(d_index)", str(d_index))))
+
+                for m_index in range(num_of_metadata_disks):
+                    disk_list.append(Conf.get(index, GconfKeys.METADATA_DISK.value.replace("(node_id)", node_id).
+                                              replace("(cvg_index)", str(cvg_index)).replace("(m_index)", str(m_index))))
+            return disk_list
+        except Exception as e:
+            Log.error(f"Unable to fetch Disk list from GConf. Error {e}")
+            raise Exception(f"Unable to fetch Disk list. Error {e}")
+
+    def get_disk_list_for_cvg(self, index, node_id, cvg_id):
+        """
+        Return list of disks for any given node and CVG.
+        Args:
+            index(str): index of conf file
+            node_id (str): node id
+            cvg_id (str): cvg id
+        Returns:
+            list: list of disks id's
+
+        >>> get_disk_list_for_cvg(index, '21d6291109304485b3daff43a06cff77', 'cvg-01')
+        ['/dev/sdd', '/dev/sde', '/dev/sdc']
+        """
+        try:
+            disk_list = []
+            cvg_list = self.get_cvg_list(index, node_id)
+            if cvg_id in cvg_list:
+                cvg_index = cvg_list.index(cvg_id)
+                num_of_data_disks = Conf.get(index, GconfKeys.DATA_COUNT.value.
+                                            replace("(node_id)", node_id).replace("(cvg_index)", str(cvg_index)))
+                num_of_metadata_disks = Conf.get(index, GconfKeys.METADATA_COUNT.value.
+                                                replace("(node_id)", node_id).replace("(cvg_index)", str(cvg_index)))
+
+                for d_index in range(num_of_data_disks):
+                    disk_list.append(Conf.get(index, GconfKeys.DATA_DISK.value.replace("(node_id)", node_id).
+                                              replace("(cvg_index)", str(cvg_index)).replace("(d_index)", str(d_index))))
+
+                for m_index in range(num_of_metadata_disks):
+                    disk_list.append(Conf.get(index, GconfKeys.METADATA_DISK.value.replace("(node_id)", node_id).
```

Comment:
[Suggestion]
As pep8 recommend, limit the line length to 79 characters.

Reply:
Yes, You're right. I already reduced the length of lines wherever possible. If still we reduce the length of lines then it will unnecessarily increase the no. of lines.

Reply:
Fine.